backend/migrate.ts

```
@@ -0,0 +1,478 @@
+    const feedItems = await db.lRange('feedItems', 0, -1);
+    const storyIds = new Set<string>();
+    
+    for (const value of feedItems) {
+        try {
+            const item = JSON.parse(value);
+            if (item.id && item.id.startsWith('story-') && !item.id.includes('+')) {
+                storyIds.add(item.id);
+            }
+        } catch (err) {
+            // Skip invalid JSON
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)